### PR TITLE
Update readme with setuptools and release

### PR DESCRIPTION
**Pull request by StefanPatman**
I updated the readme to use setuptools for installation and mention the latest release (updated automatically). Please have a look if everything is ok for merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,10 +78,27 @@ Files with extension `.gz` are uncompressed automatically
 [1]: docs/TAB_FORMAT.md
 
 ## Options
-DNAconvert uses two parsers for NEXUS format: internal and the one from python-nexus package.
+DNAconvert uses two parsers for NEXUS format: internal (default) and the one from python-nexus package.
 
-In the file `data/cfg.tab` the string of form
+In the file `data/cfg.tab` (found at the installed location) the string of form
```
Comment (necrosovereign) [inline README.md:83]: I would like to change this to `DNAconvert/config.json` in ${XDG_CONFIG_HOME} or %APPDATA%. Should I add a commit to this pull request?

Comment (StefanPatman) [reply README.md:83]: I was also worried about this, please go ahead!

File: README.md [hunk 2]
```diff
@@ -1,43 +1,26 @@
...
-Then run
+Install the latest version directly using pip (requires Python 3.8 or later):
+```
+pip install git+https://github.com/iTaxoTools/DNAconvert.git#egg=DNAconvert
+```
 
-    pyinstaller --onefile --windowed DNAconvert.py
+## Executables
+Download and run the standalone executables without installing Python.</br>
+[See the latest release here.](https://github.com/iTaxoTools/DNAconvert/releases/latest)
```
[the remark this answers is not in the document]
Comment (StefanPatman) [reply README.md:12]: I will create a new release and add the executables myself after we finalize the readme file.

File: README.md (hunk 1 above)
Comment (necrosovereign) [reply README.md]: I added the configuration handling, I think this can be merged now

**Pull request merged**